File: programmers/default_phonenumber.js
/*
	rank:	1
	title:	하샤드 수
	link: 	https://programmers.co.kr/learn/courses/30/lessons/12947
*/

function solution(phone_number) {
    return "*".repeat(phone_number.length - 4) + phone_number.substr(-4);
}

File: programmers/etc/python/default_phonenumber.py
# rank:	    1
# title:	핸드폰 번호 가리기
# link: 	https://programmers.co.kr/learn/courses/30/lessons/12948

def solution(phone_number):
    count = (len(phone_number) - 4)
    return "*" * count + phone_number[count:]